src/main/java/seedu/notus/ui/Formatter.java

```diff
@@ -210,10 +210,22 @@ public static String formatTimetable(String header, ArrayList<Event> events) {
      */
     public static ArrayList<String> formatEvent(Event event) {
         ArrayList<String> result = new ArrayList<>();
+        ArrayList<String> reminders = event.getReminderPeriodsString();
+
+        if (reminders.size() == 0) {
+            reminders.add("None");
+        }
+
+        String reminderString = "Reminders:";
+        for (String reminder : reminders) {
+            reminderString = reminderString.concat(" " + reminder);
+        }
+
```

Comment:
If there are no reminders, it will add a none, then would the output be: "None reminder"? Sounds a little weird.

Reply:
Nope! It'll display as Reminder: None